task_list_app/task_list_app/.gitignore

```diff
@@ -35,3 +35,4 @@ out/
 
 ### VS Code ###
 .vscode/
+.vscode/launch.json
```

Comment:
It's particularly the `.DS_Store` files I was mentioning. Not a deal-breaker for this pull request, but I would recommend getting rid of them in the remote repository.